contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -54,72 +27,60 @@ contract OptimismPortal {
     );
 
     /**
-     * @notice Emitted when a withdrawal is finalized
+     * Emitted when a withdrawal is finalized
      */
     event WithdrawalFinalized(bytes32 indexed, bool success);
 
-    /*************
-     * Constants *
-     *************/
-
     /**
-     * @notice Value used to reset the l2Sender, this is more efficient than setting it to zero.
+     * Value used to reset the l2Sender, this is more efficient than setting it to zero.
      */
     address internal constant DEFAULT_L2_SENDER = 0x000000000000000000000000000000000000dEaD;
 
-    /*************
-     * Variables *
-     *************/
-
     /**
-     * @notice Minimum time that must elapse before a withdrawal can be finalized.
+     * Minimum time that must elapse before a withdrawal can be finalized.
      */
-    uint256 public immutable FINALIZATION_PERIOD;
+    uint256 public immutable FINALIZATION_PERIOD_SECONDS;
 
     /**
-     * @notice Address of the L2OutputOracle.
+     * Address of the L2OutputOracle.
      */
     L2OutputOracle public immutable L2_ORACLE;
 
     /**
-     * @notice Public variable which can be used to read the address of the L2 account which
-     * initated the withdrawal. Can also be used to determine whether or not execution is occuring
-     * downstream of a call to finalizeWithdrawalTransaction().
+     * Public variable which can be used to read the address of the L2 account which initiated the
+     * withdrawal. Can also be used to determine whether or not execution is occuring downstream of
+     * a call to finalizeWithdrawalTransaction().
      */
     address public l2Sender = DEFAULT_L2_SENDER;
 
     /**
-     * @notice A list of withdrawal hashes which have been successfully finalized.
+     * A list of withdrawal hashes which have been successfully finalized.
```

Comment:
Why is `@notice` removed from all the natspec docs?

Reply:
`@notice` isn't strictly necessary and it looks ugly, we don't use notice anywhere else